### PR TITLE
Publish image to GitHub Container Registry

### PR DESCRIPTION
**Pull request by fwilhe**

### CODE REVIEW

**Comment by stippi2** [comment]
The build/test seems to depend on `stageInitS4sdkPipeline` which no longer exists.

**Comment by fwilhe** [comment]
> The build/test seems to depend on `stageInitS4sdkPipeline` which no longer exists.

Right. FYI it is in this repo https://github.com/piper-validation/cloud-s4-sdk-book, we have many branches there for individual test cases, and not all of them have been updated to breaking changes yet.

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -22,14 +22,20 @@ jobs:
...
+      run: |
+        docker tag ppiper/node-browsers:${{ matrix.base-image-tag }} ppiper/node-browsers:latest
+        docker tag ghcr.io/sap/ppiper-node-browsers:${{ matrix.base-image-tag }} ghcr.io/sap/ppiper-node-browsers:latest
     - name: Push
       if: ${{ github.ref == 'ref/head/master' }}
       run: |
         echo ${{ secrets.DOCKERHUB_PASSWORD }} | docker login -u ${{ secrets.DOCKERHUB_USER }} --password-stdin
+        echo "${{ secrets.CR_PAT }}" | docker login https://ghcr.io -u ${{ secrets.CR_USER }} --password-stdin
         docker push ppiper/node-browsers:${{ matrix.base-image-tag }}
+        docker push ghcr.io/sap/ppiper-node-browsers:${{ matrix.base-image-tag }}
```
Comment (stippi2) [inline .github/workflows/build.yml:37]: I would have expected to interleave the logins and pushes. But apparently you can be logged into two registries at the same time?

File: .github/workflows/release.yml [hunk 2]
```diff
@@ -35,13 +35,18 @@ jobs:
       - name: Build and push
         run: |
           echo ${{ secrets.DOCKERHUB_PASSWORD }} | docker login -u ${{ secrets.DOCKERHUB_USER }} --password-stdin
+          echo "${{ secrets.CR_PAT }}" | docker login https://ghcr.io -u ${{ secrets.CR_USER }} --password-stdin
           docker build --build-arg=BASE_IMAGE_TAG=${{ matrix.base-image-tag }} --tag ppiper/node-browsers:${{ env.PIPER_version }}-${{ matrix.base-image-tag }} .
+          docker tag ppiper/node-browsers:${{ env.PIPER_version }}-${{ matrix.base-image-tag }} ghcr.io/sap/ppiper-node-browsers:${{ env.PIPER_version }}-${{ matrix.base-image-tag }}
           docker push ppiper/node-browsers:${{ env.PIPER_version }}-${{ matrix.base-image-tag }}
+          docker push ghcr.io/sap/ppiper-node-browsers:${{ env.PIPER_version }}-${{ matrix.base-image-tag }}
```
Comment (stippi2) [inline .github/workflows/release.yml:42]: Same here.

File: .github/workflows/build.yml (hunk 1 above)
Comment (fwilhe) [reply .github/workflows/build.yml:37]: yes, you can be logged into multiple regs at the same time

**Review by stippi2 — APPROVED** [review]

**Pull request merged**